packages/playground/blueprints/src/lib/compile.spec.ts

```diff
@@ -151,14 +150,32 @@ describe('Blueprints', () => {
 					valid: false,
 					errors: [
 						{
-							instancePath: '/steps/0',
-							keyword: 'required',
+							instancePath: ['steps', 0],
+							keyword: 'anyOf',
 							params: {
-								missingProperty: 'themeZipFile',
+								type: {
+									anyOf: [
+										{
+											$ref: '#/definitions/StepDefinition',
+										},
+										{
+											type: 'string',
+										},
+										{
+											not: {},
+										},
+										{
+											const: false,
+											type: 'boolean',
+										},
+										{
+											type: 'null',
+										},
+									],
+								},
 							},
 							message:
-								"must have required property 'themeZipFile'",
-							schemaPath: expect.any(String),
+								'is not any of <#/definitions/StepDefinition>,[subschema 1],[subschema 2],[subschema 3],[subschema 4]',
```

Comment:
It looks like each of the "anyOf" option descriptions could be made more useful with a `title` property. Check out the jsonschema code that generates the strings like "[subschema 2]":
https://github.com/tdegrunt/jsonschema/blob/84ed77ba2e5f9f3f4dbd340499a7f4efcaf1deb5/lib/attribute.js#L103

Reply:
If it is more important to keep the try-wordpress work rolling (and it might be), let's at least log the opaque error messages as a Blueprint UX issue before merging this PR.